climada_petals/hazard/wildfire.py

```diff
@@ -25,24 +25,42 @@
 import itertools
 from dataclasses import dataclass
 from datetime import date
+from pathlib import Path
+import copy
 
 import numpy as np
 import matplotlib.pyplot as plt
 import pandas as pd
 from scipy import sparse
+import scipy.ndimage # added by Sam G.
```

Comment:
This is unrelated to this line, but on line 36 you import a ` Balltree`. Is there a specific reason to do so? If it is to match centroids, I would suggest to use the already implemented methods in the `util.coordinates`. The speed of the method can drastically change when not using the otimized methods.